plugins/accuracy.cc

```diff
@@ -277,14 +285,31 @@ runCategorical(AccuracyConfig & runAccuracyConf,
                         outputRow.emplace_back(score + columnName, v, recordDate);
                     }
 
+                    if (accuracyOverN > 1) {
+                        bestlabelsCandidates.emplace_back(v, 
+                            CellValue(jsonDecodeStr<CellValue>(columnName.toSimpleName())));
+                    }
+
                     return true;
                 };
             scoreLabelWeight[0].forEachAtom(onAtom);
 
-            auto label = scoreLabelWeight[1].getAtom();
+            if (bestlabelsCandidates.size() > accuracyOverN) {
+                std::sort(bestlabelsCandidates.begin(), 
+                          bestlabelsCandidates.end(), 
+                          std::greater<std::pair<double, CellValue>>());
+                bestlabelsCandidates.erase(bestlabelsCandidates.begin()+accuracyOverN, bestlabelsCandidates.end());
```

Comment:
Big problem here: you aren't handling ties properly.  This will cause problems with reproducibility.  If there are 5 values tied for 3rd place, then you need to give 1/5 of a point for the top 3.  This section needs to be redone.